processor/spanmetricsprocessor/config.go

```diff
@@ -15,12 +15,16 @@
 package spanmetricsprocessor // import "github.com/open-telemetry/opentelemetry-collector-contrib/processor/spanmetricsprocessor"
 
 import (
+	"fmt"
 	"time"
 
+	"go.opentelemetry.io/collector/component"
 	"go.opentelemetry.io/collector/featuregate"
 	"go.opentelemetry.io/collector/pdata/pmetric"
 )
 
+var _ component.ConfigValidator = (*Config)(nil)
```

Comment:
Please move to line 75 so that it appears after type definition to improve readability.